Monika After Story/game/event-handler.rpy

```diff
@@ -1977,6 +1977,9 @@ label call_next_event:
                 $ mas_clearNotifs()
                 jump _quit
 
+            if "love" in ret_items:
```

Comment:
return items should be checked before the `quit` since that will jump out from here. move this check to before the `quit` one